docs/usage/data-modeling.md

```diff
@@ -0,0 +1,150 @@
+# Data Modeling
+
+TigerBeetle is a domain-specific database — its schema of [`Account`s](../reference/accounts.md)
+and [`Transfer`s](../reference/transfers.md) is built-in and fixed. In return for this prescriptive
+design, it provides excellent performance, integrated business logic, and powerful invariants.
+This section is a sample of techniques for mapping an application's requirements onto TigerBeetle's
+data model. Which (if any) of these techniques are suitable is highly application-specific.
+
+When possible, encoding application invariants directly in TigerBeetle rather than implementing
+them in the application itself (or with a foreign database) minimizes round-trips and coordination.
+This is useful for both maintaining consistency and performance.
+
+## Debits vs Credits
+
+In double-entry accounting, an account balance is either computed as either `debits - credits` or
+`credits - debits`, depending on the type of account. It is up to the application to compute
+the balance — TigerBeetle tracks the cumulative posted debits and cumulative posted credits,
+not their difference.
```

Comment:
For a second this makes it sound like the application must sum up all transfers themselves.

Maybe instead:

> TigerBeetle tracks the cumulative posted debits and cumulative posted credits, not their difference. If the application needs the difference, it can be easily computed from the cumulative debits and credits tracked on the account.

Or something?

Reply:
I changed it to:

> TigerBeetle tracks each account's cumulative posted debits and cumulative posted credits.
In double-entry accounting, an account balance is the difference between the two — computed as
either `debits - credits` or `credits - debits` depending on the type of account. It is up to the
application to compute the balance from the cumulative debits/credits.

Reply:
`compute the balance` still sounds a little scary but it's clearer!